after/ftplugin/query.vim

```diff
@@ -0,0 +1 @@
+setlocal omnifunc=v:lua.require'nvim_treesitter_playground_query_omnifunc'.omnifunc
```

Comment:
Neovim does not allow `-` in require. This might be a bug...

Reply:
It's indeed a bug!